client/src/pages/SignUp/SignUp.js

```diff
@@ -92,7 +92,6 @@ class SignUp extends Component {
                 value={this.state.username}
                 onChange={this.handleChange}
               />
-              {console.log(this.state)}
```

Comment:
We should think about removing console logs in the application. Console logs are really only used for debugging, not for production.